doc/nixos/tutorials/archiver-appliance.md

```diff
@@ -0,0 +1,338 @@
+  services.archiver-appliance.stores.mts.location = "/tmp/mts";
+}
+```
+
+If you don't want to repeat yourself,
+you can also change it like so:
+
+``` {.nix filename="archiver-appliance.nix"}
+{
+  services.archiver-appliance = {
+    enable = true;
+    stores.lts.location = "/tmp/lts";
+    stores.mts.location = "/tmp/mts";
+  };
+}
+```
+
+And now,
+`nixos-rebuild test` should succeed:
+
+::: sourceCode
+``` sourcecode
+building the system configuration...
+activating the configuration...
+setting up /etc...
+reloading user units for admin...
+setting up tmpfiles
+reloading the following units: dbus.service
+the following new units were started: arch-lts-ArchiverStore.mount,
+  arch-mts-ArchiverStore.mount, arch-sts-ArchiverStore.mount,
+  mysql.service, tomcat.service
+```
+:::
+
+From the message,
+we can guess it started the Tomcat server running Archiver Appliance,
+the MySQL (in fact, MariaDB) server,
+and some mounted some partitions.
+Fantastic!
+
+You can run the `systemctl` command to see if any systemd unit failed.
+
+In the default configuration,
+Archiver Appliance and Tomcat are configured to output logs to journald.
+You can see those logs by running:
+
+``` bash
+journalctl -xeu tomcat.service
+```
+
+::: callout-note
+Here are some details on what was done by EPNix' `services.archiver-appliance` NixOS module:
+
+-   Creation of the Linux user and group `archappl`
+-   Installation and configuration of MariaDB:
+    -   Creation of the `archappl` user,
+        with UNIX socket authentication
+    -   Creation of the Archiver Appliance database
+    -   Creation of the [various tables] in that database
+    -   Giving access rights to this database for the `archappl` user
+-   Installation and configuration of Tomcat:
+    -   Installation of the WAR files of Archiver Appliance
+    -   Installation of the MariaDB connector and its dependencies
+    -   Configuring the MariaDB connector to authenticate to the database
+    -   Logging configuration to `journald`
+-   Configuring mounts so that:
+    -   `/arch/lts` and `/arch/mts` are bind mounts to the configured locations,
+        with some added security options,
+        such as `nodev` and `noexec`
+    -   Mounting `/arch/sts` as a new `tmpfs`
+:::
+
+Tomcat runs by default under port 8080,
+and NixOS has a firewall enabled by default.
+
+Change your `archiver-appliance.nix`:
+
+``` {.nix filename="archiver-appliance.nix"}
+{
+  services.archiver-appliance = {
+    enable = true;
+    stores.lts.location = "/tmp/lts";
+    stores.mts.location = "/tmp/mts";
+
+    # New option:
+    openFirewall = true;
+  };
+}
+```
+
+and run `nixos-rebuild test`.
+It will restart `firewall.service`,
+but configured to allow incoming connection on port 8080.
+
+Check the IP address of your VM with `ip a`,
+and open a browser to `http://<YOUR_VM_IP>:8080/mgmt/ui/index.html`.
+
+You have now configured Archiver Appliance on NixOS.
```

Comment:
You're right I should instruct the user to run `nixos-rebuild switch` to "validate" the changes, and make it so that the modifications are valid next boot.